### PR TITLE
Explicitly bind to all interfaces; update README

### PR DESCRIPTION
**Pull request by gravityrail**
Just a couple of small tweaks I made when trying to get it to work for me.

### CODE REVIEW

File: lighthouse_machine/README.md [hunk 1]
```diff
@@ -9,16 +9,28 @@ docker build --no-cache -t lighthouse_machine .
...
+
+```bash
+curl -X GET 'https://localhost:8443?format=${format}&url=${url}' --insecure
+```
+
 where `format`is one of `json`, `html` (see [cli-options](https://github.com/GoogleChrome/lighthouse#cli-options) for more information)
 
+## FAQ
+
+Q: I get an "Empty response" frmo the server. What's happening?
```
Comment (juliantoledo) [inline lighthouse_machine/README.md]: typo s/frmo/from

**Comment by andreban** [comment]
Hi, @gravityrail, do you plan to update the PR? Otherwise, let us know and we'll assign to someone else.

**Comment by gravityrail** [comment]
@andreban updated to fix the typo. Merge when ready!

**Comment by gravityrail** [comment]
Oh and I was unsuccessful in signing the contributor agreement, sorry. Let me know if there are any workarounds, this was a while ago so I can't remember the exact details but I tried and couldn't get it to work.

**Comment by andreban** [comment]
@gravityrail Can you see the correct information when checking https://cla.developers.google.com/clas?

**Comment by gravityrail** [comment]
@andreban yes the problem was some issue with the Google Groups stuff, if I recall. I've never had any luck with it using my work email. We are on gsuite.

**Comment by andreban** [comment]
Gotcha. In this case, I think it needs to be a corporate CLA, which seems that your company has  already signed. 

You need to be added at the proper internal google group in your company. There's more info here: https://cla.developers.google.com/about#add-contributors.

Can you confirm if all those steps have been performed?

Sorry, I know it's a bit bureaucratic. We are really grateful for the PR, be we can only accept it with the CLA in place.

Thanks, 
André

**Pull request closed without merging**